sensorserviceutils/sensorserviceutils.go

```diff
@@ -1089,6 +1094,30 @@ func (p *parser) extractRegistrationHistory() []error {
 	return p.parsingErrs
 }
 
+// createHistoryForEventsWithNoActivation is a function that create history
+// for sensor activities such that deactivation statement is present but no 
+// activation statement is seen in the sensor dump history. 
+// The visualizer will show the event as it starts
+// when the sensor history first starts.
```

Comment:
nit: You can do in future commit

the line break is too early. You can move "when the sensor" to previous line.